src/NIDAQ.jl

```diff
@@ -42,6 +42,8 @@ export analog_input_channels, analog_output_channels
 export digital_input_channels, digital_output_channels
 export counter_input_channels, counter_output_channels
 
+export TerminalConfig, RSE, NRSE, Differential, PseudoDifferential
```

Comment:
will `TerminalConfig` ever need to be referenced by the user?  if not, perhaps don't export it.

Reply:
Will roll this in tomorrow when I get to work